source/includes/api/catalog.md

```diff
@@ -1,6 +1,6 @@
 # Products catalog
 
 Once your account created and configured (follow the onboarding process for that), you must push your products catalog.
-Only product in the ShopRunBack catalog can be returned.
+Only products in the ShopRunBack catalog can be returned.
 
-First, create your brands if you have any and then create your products.
\ No newline at end of file
+First, create your brand(s) if you don't have any and then create your products.
```

Comment:
no, it is not required to create brand. With your update, the user will think that he must create brands.